src/microscope_tiling_basler_camera/src/microscope_tiling_basler_camera/path_builder.py

```diff
@@ -0,0 +1,114 @@
+"""PathBuilder module"""
+
+import numpy as np
+from numpy.typing import NDArray
+from zaber_motion import Units
+from example_util import (
+    convert_length_to_microns,
+    convert_point_to_microns,
+)
+
+
+class PathBuilder:
+    """
+    PathBuilder provides functionality for generating paths for
+    camera tiling
+    """
+
+    Path = list[list[tuple[float, float]]]
```

Comment:
Minor suggestion: rename this `MotionPath` instead of just `Path` because `Path` is an often-used class in a popular python library called `pathlib`.  Even though your path is in its own namespace, when I read it I immediately think of OS path instead.